Makefile

```diff
@@ -300,6 +305,17 @@ installoptopt:
 	cd $(INSTALL_COMPLIBDIR) && $(RANLIB) ocamlcommon.a ocamlbytecomp.a \
 	   ocamloptcomp.a
 
+# Installation of the *.ml sources of compiler-libs
+install-compiler-sources:
+	cp utils/*.ml parsing/*.ml typing/*.ml bytecomp/*.ml driver/*.ml \
+	   toplevel/*.ml $(INSTALL_COMPLIBDIR)
+	if test -f ocamlopt; then $(MAKE) installopt-compiler-sources; fi
+
+# Installation of the *.ml sources native-code part of compiler-libs
+installopt-compiler-sources:
```

Comment:
Why not merge these two targets ? If somebody wants access to the compiler sources, he probably wants access to both bytecode and native code compilers.